GitUI/GitUICommands.cs

```diff
@@ -372,6 +372,26 @@ bool Action()
             return DoActionOnRepo(owner, true, true, null, null, Action);
         }
 
+        public bool StartShowBranchDialog(ref FormCreateBranch formCreateBranch, IWin32Window owner = null)
+        {
+            var form = formCreateBranch;
+            formCreateBranch = null;
+            bool Action()
+            {
+                var result = form.ShowDialog(owner) == DialogResult.OK;
+                return result;
+            }
+
+            try
+            {
+                return DoActionOnRepo(owner, true, true, null, null, Action);
+            }
+            finally
+            {
+                form.Dispose();
```

Comment:
Would `using (form)` be simpler here? Neighbours tend to use `using` statements.

Reply:
`using (value)` is a valid alternative to `using (declaration)`.

The form that accepts a declaration is a special case. Usually a declaration (`var a = 1`) doesn't produce a value (it's not an expression). An assignment does though, so you can write `var i = (j = 0)` and `i` will be zero too.